### PR TITLE
Ft update buyer cart

### PR DESCRIPTION
**Pull request by tolls-3**
## Description
Describe change or new feature here.
Create an endpoint for buyers to update products in their cart. 
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/ZmfuayqP/93-api-endpoint-put-buyer-cart

### CODE REVIEW

**Comment by tolls-3** [comment]
[API Endpoint: PUT buyer cart](https://trello.com/c/ZmfuayqP/93-api-endpoint-put-buyer-cart)

File: routes/cartRouter.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
 // @route POST api/store/:store_id/cart
 // @desc  Add items to cart
 // @access Public
 router.post('/:store_id/cart', addCart)
 
+// @route PUT api/store/:store_id/cart
+// @desc  Add items to cart
+// @access Public
+router.put('/cart/:cart_id', editCart)
```
Comment (aapetsi) [inline routes/cartRouter.js:15]: confirm the route naming. is it supposed to be `/cart/:cart_id` or `/store/:cart_id`

Comment (tolls-3) [reply routes/cartRouter.js:15]: This is the full route `/api/store/cart/:cartId` . Let me know what you think, we can always change it if needed.

Comment (aapetsi) [reply routes/cartRouter.js:15]: then that's fine. i didn't check the full route

**Review by aapetsi — APPROVED** [review]

**Pull request merged**